ga4gh/datamodel/__init__.py

```diff
@@ -229,6 +256,8 @@ class VariantSetCompoundId(DatasetCompoundId):
     """
     fields = DatasetCompoundId.fields + ['variantSet']
     containerIds = DatasetCompoundId.containerIds + [('variantSetId', 1)]
+    obfuscators = [DatasetCompoundId, None]
```

Comment:
The only thing the type-specific obfuscator really does is point to the correct prefix to obfuscate with for that type.